lib/msal-common/src/client/BaseClient.ts

```diff
@@ -54,7 +59,24 @@ export abstract class BaseClient {
 
         B2cAuthority.setKnownAuthorities(this.config.authOptions.knownAuthorities);
 
-        this.defaultAuthority = this.config.authOptions.authority;
+        this.authority = this.config.authOptions.authority;
+    }
+
+    /**
+     * Use to log out the current user, and redirect the user to the postLogoutRedirectUri.
+     * Default behaviour is to redirect the user to `window.location.href`.
+     * @param authorityUri
+     */
+    async getLogoutUri(logoutRequest: EndSessionRequest): Promise<string> {
```

Comment:
This is only applicable to SPA, so I don't it should live in BaseClient.

Reply:
This could be a convenience function for msal-node in the future, I will keep it here for now.

Reply:
Right, although it could be a convenience function, it's still not applicable to 7/8 flows, so I don't why it would be here.

Reply:
If it's ever used for Node, it would also just be used for authorization code flow

Reply:
OK I will move it to Auth code client

Reply:
done.